tests/README.md

```diff
@@ -24,5 +24,7 @@ Following parameters can be configured as a arguments to test:
 Available Integration Tests:
 | Test Name       | Expected behavior | Test Link  |
 | --------------- | ----------------- | ---------- |
-| Create NFS PVC with reclaim policy Delete | <ul> Verify whether volume_event_controller sent volume_provisioned JSON data to configured server <li> nfs_pvc </li> <li> nfs_pv </li> <li> backend_pvc </li> <li> backend_pv </li> along with creationTimestamp on `nfs_pv` </ul>| [nfs_sanity_test.go](./nfs_sanity_test.go) |
-| Delete NFS PVC with reclaim policy Delete | <ul> Verify whether volume_event_controller sent volume_deprovisioned JSON data to configured server <li> nfs_pv </li> <li> backend_pvc </li> <li> backend_pvc </li> along with creation and deletion timestamp on `nfs_pv` </ul> | [nfs_sanity_test.go](./nfs_sanity_test.go) |
+| Create NFS PVC with reclaim policy Delete | <ul> <li> Verify whether volume_event_controller sent volume_provisioned JSON data to configured server <ul> <li> nfs_pvc </li> <li> nfs_pv </li> <li> backend_pvc </li> <li> backend_pv </li> along with creationTimestamp on `nfs_pv` </ul> </ul> </li>| [nfs_sanity_test.go](./nfs_sanity_test.go) |
+| Delete NFS PVC with reclaim policy Delete | <ul> <li> Verify whether volume_event_controller sent volume_deprovisioned JSON data to configured server <ul> <li> nfs_pv </li> <li> backend_pvc </li> <li> backend_pvc </li> along with creation and deletion timestamp on `nfs_pv` </ul> </ul> </li> | [nfs_sanity_test.go](./nfs_sanity_test.go) |
+| Create NFS PVC with ReclaimPolicy Retain | <ul> <li> Verify whether volume_event_controller sent volume_provisioned JSON data to configured server <ul> <li> nfs_pvc </li> <li> nfs_pv </li> <li> backend_pvc </li> <li> backend_pv </li> along with creationTimestamp on `nfs_pv` </ul> </ul> </li> | [nfs_pvc_retain_test.go](./nfs_pvc_retain_test.go) |
+| Delete NFS PVC with ReclaimPolicy Retain | <ul> <li> volume_event_controller shouldn't send event when `nfs_pvc` is deleted </li> <li> Verify whether volume_event_controller sent volume_provisioned JSON data to configured server upon `nfs_pv` deletion <ul> <li> nfs_pv </li> <li> backend_pvc </li> <li> backend_pv </li> along with creation and deletion timestamp on `nfs_pv` </ul> </ul> </li> | [nfs_pvc_retain_test.go](./nfs_pvc_retain_test.go) |
```

Comment:
let's add a separate entry for `Destroy NFS PV with ReclaimPolicy Retain`

Reply:
Updated